### PR TITLE
Teams fx migration samples

### PR DESCRIPTION
**Pull request by Harikrishnan-MSFT**

### CODE REVIEW

File: samples/app-checkin-location/nodejs/appPackage/manifest.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ﻿{
-  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.12/MicrosoftTeams.schema.json",
-  "manifestVersion": "1.12",
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.13/MicrosoftTeams.schema.json",
```
Comment (Prithvi-MSFT) [inline samples/app-checkin-location/nodejs/appPackage/manifest.json:2]: remove manifest zip file

**Review by Prithvi-MSFT — COMMENTED** [review]
provided review comments

File: samples/app-checkin-location/nodejs/appPackage/manifest.json (hunk 1 above)
Comment (Harikrishnan-MSFT) [reply samples/app-checkin-location/nodejs/appPackage/manifest.json:2]: Already deleted.
FYI
image

**Review by Prithvi-MSFT — APPROVED** [review]
changes look fine

**Comment by ChetanSharma-msft** [comment]
@Harikrishnan-MSFT - Please run build pipeline and share the result.

**Comment by Harikrishnan-MSFT** [comment]
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/5641334002
image

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

**Pull request merged**